src/components/Proposal/ProposalSummary/ProposalSummaryPluginManager.tsx

```diff
@@ -101,7 +101,7 @@ class ProposalSummary extends React.Component<IProps, IState> {
                 <div className={css.summaryDetails}>
                   <table>
                     <tbody>
-                      {isReplace ? 
+                      {isReplace ?
```

Comment:
why this file has been changed ? 
is that intentionally , removing white spaces ?

Reply:
Somehow the linter showed me an error about this.